test/e2e/config/e2e_conf.yaml

```diff
@@ -87,7 +83,8 @@ variables:
   CAPIRELEASE: "${CAPIRELEASE}"
   CAPM3RELEASE: "${CAPM3RELEASE}"
   CONFIG_FILE_PATH: "${HOME}/.cluster-api/overrides/clusterctl.yaml"
-
+  # Default vars for the template, those values could be overridden by the env-vars.
+  SERVICE_CIDR: "10.96.0.0/12"
```

Comment:
I do not think it is exported in the env because e2e was complaining about this var while generating the template but I can add possibility to read from env else use the default value

Reply:
Well I guess e2e will prioritize the value in the env else it use what is declared in conf file so no need to read from env in the conf file